Makefile

```diff
@@ -119,6 +119,25 @@ clean: .cleanpycs .cleandocs
 	@echo "Removing generated documentation"
 	rm -rf $(DOC_BUILD_DIR)
 
+.PHONY: .cleanmongodb
+.cleanmongodb:
+	@echo "==================== cleanmongodb ===================="
+	@echo "----- Dropping all MongoDB databases -----"
+	@sudo service mongodb force-stop
+	@sudo rm -rf /var/lib/mongodb/*
```

Comment:
That's obviously only meant to be used in a local development environment :)

I remove the data dir, because just removing a database, doesn't reclaim the disk space (you need to run a compaction, etc.).